money-rails.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
 
   s.require_path = "lib"
 
-  s.add_dependency "money",         "~> 6.5.0"
+  s.add_dependency "money",         "~> 6.8.2"
   s.add_dependency "monetize",      "~> 1.1.0"
```

Comment:
I got the build working by bumping this to 1.6.0. I can't run tests though, on this branch or master 😕